modules/capi-modifying-machine-template.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-managing-machines.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="capi-modifying-machine-template_{context}"]
+= Modifying a Cluster API machine template
+
+You can update the machine template resource for your cluster by modifying the YAML manifest file and applying it with the {oc-first}.
+
+.Prerequisites
+
+* You have deployed an {product-title} cluster that uses the Cluster API.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+.Procedure
+
+. List the machine template resource for your cluster by running the following command:
++
+--
+[source,terminal]
+----
+$ oc get <machine_template_kind> <1>
+----
+<1> Specify the value that corresponds to your platform. The following values are valid:
+* `AWSMachineTemplate`: The cluster is running on {aws-first}.
+* `GCPMachineTemplate`: The cluster is running on {gcp-first}.
+--
++
+.Example output
+[source,text]
+----
+NAME              AGE
+<template_name>   77m
+----
+
+. Write the machine template resource for your cluster to a file that you can edit by running the following command:
++
+[source,terminal]
+----
+$ oc get <template_name> -o yaml > <template_name>.yaml
+----
+
+. Use a text editor to make changes to the machine template resource for your cluster.
+When editing the machine template resource, observe the following:
+
+** The parameters in the `spec` stanza are provider specific.
+For more information, see the sample Cluster API machine template YAML for your provider.
+
+** You must use a value for the `name` parameter that differs from the existing value.
++
+[IMPORTANT]
+====
+You must update any Cluster API compute machine sets in your cluster to reference the `name` value in the new machine template resource.
```

Comment:
Not all MS will have the same template, so need to make this clear that it's appropriate/desired MachineSets that get an update